### DISCUSSION TITLE
Can't open search field & menus

### DISCUSSION THREAD

**Discussion by 0123456789-jpg** [community/community#109687] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I recently found that when i click the search field & menus (marked as red below) it just does nothing.
<img width="1298" alt="Screenshot 2024-02-23 20 40 56" src="https://github.com/community/community/assets/56025753/5aec91c5-27c0-4ba4-9825-2d4263c4b966">
I checked the console and it said `Unhandled Promise Rejection: ReferenceError: Can't find variable: HTMLDialogElement` when i clicked. I checked out the MDN and it says `HTMLDialogElement` is supported in Safari 15.4 and newer, but the newest version i can get is Safari 14.x (ps. i only use github with safari). I think github should have an option to let users to switch to the old impl for compatibility.
If there's already one please tell me how to switch it. I can't open beta features panel because menu doesn't work.

**Comment by voss01** [comment]
same here
